pkgs/tools/misc/webcat/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchgit }:
+
+buildGoModule rec {
+  pname = "webcat";
+  version = "unstable-20210906";
+
+  src = fetchgit {
+    url = "https://codeberg.org/rumpelsepp/webcat.git";
+    rev = "57a65558f0affac0b2f8f4831c52964eb9ad5386";
+    sha256 = "15c62sjr15l5hwkvc4xarfn76341wi16pjv9qbr1agaz1vqgr6rd";
+  };
```

Comment:
`fetchFromGitea` might be better here

Reply:
You are right, I will use `fetchFromGitea`

Reply:
Done